platforms/web/test-e2e/tests/payments/payments_test.ts

```diff
@@ -0,0 +1,228 @@
+import { testConfigs } from '@jwp/ott-testing/constants';
+
+import { LoginContext } from '#utils/password_utils';
+import constants, { longTimeout } from '#utils/constants';
+import { goToCheckout, finishSubscription, cancelPlan, renewPlan, overrideIP, addYear, formatDate, checkSubscription } from '#utils/payments';
+import { ProviderProps } from '#test/types';
+
+const jwProps: ProviderProps = {
+  config: testConfigs.jwpSvod,
+  monthlyOffer: constants.offers.monthlyOffer.inplayer,
+  yearlyOffer: constants.offers.yearlyOffer.inplayer,
+  paymentFields: constants.paymentFields.inplayer,
+  creditCard: constants.creditCard.inplayer,
+  applicableTax: 0,
+  canRenewSubscription: false,
+  canOpenReceipts: false,
+  fieldWrapper: '',
+  hasInlineOfferSwitch: true,
+};
+
+const cleengProps: ProviderProps = {
+  config: testConfigs.svod,
+  monthlyOffer: constants.offers.monthlyOffer.cleeng,
+  yearlyOffer: constants.offers.yearlyOffer.cleeng,
+  paymentFields: constants.paymentFields.cleeng,
+  creditCard: constants.creditCard.cleeng,
+  applicableTax: 21,
+  canRenewSubscription: true,
+  canOpenReceipts: false, // Cleeng returns an error on Sandbox making this test flaky
```

Comment:
`constants.canShowReceipts` is now `false` by default for Cleeng (not supported), so I think we can remove this comment.

Reply:
I removed the setting and test